portal-impl/src/portal.properties

```diff
@@ -8362,7 +8361,6 @@
         WEB-INF/lib/portal-kernel.jar,\
         WEB-INF/lib/portlet.jar,\
         WEB-INF/lib/portletmvc4spring-framework.jar,\
-        WEB-INF/lib/rome.jar,\
```

Comment:
@dantewang Is it safe to remove without adding the new ones?

Reply:
@tinatian rome's replacement rometools is already in portal.